build.gradle

```diff
@@ -41,7 +41,7 @@ subprojects {
     return
   }
 
-  version = '4.1.7'
```

Comment:
Please don't add snapshot to the version

Reply:
4.1.7 will be the next version to be released

Reply:
Thx, changed back to 4.1.7